docs/Overview.rst

```diff
@@ -7,3 +7,23 @@ H2020 Project E-Land
 
 License
 -------
+
+Flowchart
+-------
+ 
+The MVS’ global flowchart, or graphical model, is divided into three connected blocks that trace the logic sequence: inputs, system model, and outputs. This is a typical representation of a simulation model.
```

Comment:
```suggestion
The MVS' global flowchart, or graphical model, is divided into three connected blocks that trace the logic sequence: Inputs, system model, and outputs. This is a typical representation of a simulation model.
```

Reply:
why do you want the 'i' in inputs as a capital letter?

Reply:
Haha, probably because it is german spelling... suppose I always am mispelling stuff in english?